test/TestRulesCollection.py

```diff
@@ -92,3 +94,21 @@ def test_no_duplicate_rule_ids(test_rules_collection):
     real_rules = RulesCollection([os.path.abspath('./lib/ansiblelint/rules')])
     rule_ids = [rule.id for rule in real_rules]
     assert not any(y > 1 for y in collections.Counter(rule_ids).values())
+
+
+def test_rich_rule_listing():
+    rules_path = os.path.abspath('./test/rules')
+    command = [sys.executable, "-m", "ansiblelint", "-r", rules_path, "-f", "rich", "-L"]
```

Comment:
Don't we have a fixture helper for this already?

Reply:
Not a compatible one. The one we have works only with paths.

We could change it to also cover for this use but i did not want to increase the scope of this change. It is better to do that part as a standalone change.